### PR TITLE
fix: Account Element - Menu SVG icons support.

### PR DESCRIPTION
**Pull request by premanshup**
### Description

https://trello.com/c/PGLpeYu2/641-package-testing

### Screenshots

https://a.cl.ly/5zuBgXre

### Types of changes

Bug fix

### Checklist:
- [ ] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [x] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/extras.php [hunk 1]
```diff
@@ -390,7 +390,7 @@ function astra_remove_controls( $wp_customize ) {
 function astra_dropdown_icon_to_menu_link( $title, $item, $args, $depth ) {
 	$role     = 'presentation';
 	$tabindex = ' tabindex="0"';
-	$icon     = defined( 'ASTRA_EXT_VER' ) && Astra_Ext_Extension::is_active( 'nav-menu' ) ? '' : Astra_Icons::get_icons( 'arrow' );
+	$icon     = defined( 'ASTRA_EXT_VER' ) && Astra_Ext_Extension::is_active( 'nav-menu' ) && ( isset( $args->container_class ) && 'account-main-header-bar-navigation' !== $args->container_class ) ? '' : Astra_Icons::get_icons( 'arrow' );
```
Comment (Nikschavan) [inline inc/extras.php:393]: This condition has increased quite a lot now - maybe it makes sense to part away from the ternary condition?

File: inc/extras.php [hunk 2]
```diff
@@ -390,7 +390,7 @@ function astra_remove_controls( $wp_customize ) {
...
 	foreach ( $item->classes as $value ) {
 		if ( 'menu-item-has-children' === $value ) {
 			$title = $title . '<span role="' . $role . '" class="dropdown-menu-toggle"' . $tabindex . '>' . $icon . '</span>';
```
Comment (Nikschavan) [inline inc/extras.php:396]: While not part of this PR - Looks like `$role` and other attributes should be escaped?

File: inc/extras.php (hunk 1 above)
Comment (premanshup) [reply inc/extras.php:393]: Agree! making changes to it.

Comment (premanshup) [reply inc/extras.php]: @Nikschavan Done!

File: inc/extras.php (hunk 2 above)
Comment (premanshup) [reply inc/extras.php]: @Nikschavan Right! I have added esc_attr to the attributes.
